package.json

```diff
@@ -156,7 +156,7 @@
     "rollup-plugin-visualizer": "^4.2.0",
     "semantic-release": "^18.0.0",
     "semantic-release-cli": "^5.4.4",
-    "stream-chat": "^5.0.0",
+    "stream-chat": "6.0.0",
```

Comment:
Should not we keep the `^` here?

Reply:
Agreed, I will push the change. Thanks for reporting. I missed it out while upgrading the package.